nvidia-container-toolkit/glibc/pkg.yaml

```diff
@@ -13,6 +13,7 @@ steps:
       DEBIAN_FRONTEND: noninteractive
     prepare:
       - |
+        chmod 1777 /tmp
```

Comment:
bldr issue?

Reply:
yep, exactly, it resets permissions on merge (?). not sure why?

but seems harmless otherwise